clicache/src/impl/PdxReaderWithTypeCollector.hpp

```diff
@@ -80,19 +80,19 @@ namespace Apache
           /// Read a 16-bit unsigned integer from the stream.
           /// </summary>
           /// <param name="fieldName">The name of a member field whose value to read.</param>
-          virtual System::UInt16 ReadUInt16(String^ fieldName)override;
+          virtual System::UInt16 ReadUShort(String^ fieldName)override;
```

Comment:
Insert standard naming rant here.  This one is more understandable, in that it was inconsistent in the original, but let's not lose any information in the process.

Reply:
Same as above. Microsoft uses the C# keywords in their API docs.